src/main/java/net/masterthought/cucumber/ReportParser.java

```diff
@@ -93,7 +94,7 @@ private Feature[] parseForFeature(String jsonFile) {
             }
             String jsonFileName = extractQualifier(jsonFile);
             Arrays.stream(features).forEach(feature ->
-                    feature.setQualifier(StringUtils.defaultString(configuration.getQualifier(jsonFileName), jsonFileName))
+                    feature.setQualifier(Objects.toString(configuration.getQualifier(jsonFileName), jsonFileName))
```

Comment:
would have bigger sense if the import of unused class could be deleted or even the whole dependency can go out of pom.xml file

Reply:
I agree, the issue was it was deprecated, so that was the reason there.  However, to try to show direction better, added a few more adjustments here, renamed the PR, and showing that intent is to eventually drop commons* in general as that is really for java 7 or before rather than java 8+ or as repo was just switched java 11+ now.